asteroidsyncservice/watch.cpp

```diff
@@ -89,3 +101,14 @@ void Watch::setTime(QDateTime t)
 {
     m_iface->call("SetTime", t.currentDateTime());
 }
+
+bool Watch::timeServiceReady()
+{
+    return m_timeServiceReady;
+}
+
+void Watch::tServiceReady()
+{
+    m_timeServiceReady = true;
+    emit timeServiceChanged();
```

Comment:
If this function gets called twice in a row for whatever reason and the value hasn't actually changed, then the "changed" signal is emitted. It would be good to check if the value has changed

Reply:
Is it best to give the function a boolean parameter?